VMEncryption/main/oscrypto/rhel_81/encryptstates/PatchBootSystemState.py

```diff
@@ -107,6 +107,8 @@ def should_exit(self):
             sleep(5)
             # the restarted vm shall see the marker and advance the state machine
             self.command_executor.Execute('reboot')
+            # reboot race condition sleep
```

Comment:
what is the race here? Is that 5 seconds wait to delay reboot?
Are there other threads that should finish?

Reply:
It seemed that the reboot was sent, but some lines of code were being executed before the actual reboot happened. Sort of odd, but now, a sleep would execute rather than whatever next process was